include/dgl/aten/coo.h

```diff
@@ -195,6 +220,20 @@ bool COOHasDuplicate(COOMatrix coo);
  */
 std::pair<COOMatrix, IdArray> COOCoalesce(COOMatrix coo);
 
+/*!
+ * \brief Sort the indices of a COO matrix in-place.
+ *
+ * The function sorts row indices in ascending order. If sort_column is true,
+ * col indices are sorted in ascending order too. The data array of the returned COOMatrix
+ * stores the shuffled index which could be used to fetch edge data.
+ *
+ * Complexity: O(N*log(N)), where N is the number of nonzeros.
```

Comment:
The complexity should be lower than this.

Reply:
It should be O(N). On GPU, we use cusparse's sort API so it should be fine. On CPU, we currently use an NlogN sort. Since COOSort on CPU is not used in any critical path, I left a TODO for future improvement.